liby2/src/Y2SerialComponent.cc

```diff
@@ -344,13 +344,13 @@ bool Y2SerialComponent::initializeConnection()
 
 	 // first write one space so the other side gets
 	 // fullfilled its necessities
-	 write(fd_serial, space_buf, 1);
+	 if(write(fd_serial, space_buf, 1));
```

Comment:
same for all changes in this file. If we read return value, we should not silently ignore it.